src/main/java/org/corfudb/infrastructure/log/RollingLog.java

```diff
@@ -39,10 +36,17 @@
 public class RollingLog extends AbstractLocalLog {
 
     private Map<Long, FileHandle> channelMap;
+    private String pathSeparator;
 
     public RollingLog(long start, long end, String path, boolean sync) {
         super(start, end, path, sync);
         channelMap = new HashMap<>();
+        if (File.pathSeparator.equals(":")) {
+            pathSeparator = "/";     // OS X ... use slash, please
```

Comment:
So, the comment should say,

`// in OS X, : is the default pathSeparator, so force it to UNIX-standard "/". `

See how that makes the intent of your code clear through your comment, in a concise manner versus the current comment, which is asking OS X to do something?

But this doesn't make sense, pathSeparator is used to separate paths in environment variables. It is ":" in **ALL** UNIX systems. I think you meant to use File.Separator. But why not just use File.Separator instead of creating this new pathSeparator variable?